robot/description/urdf/constants.xacro

```diff
@@ -18,14 +21,67 @@
     <xacro:property name="caster_link_to_wheel_dist_z" value="-0.04361749"/>
 
     <!-- Wheels -->
+    <!-- TODO: update parameters once they become known -->
     <xacro:property name="wheel_vertical_offset" value="-0.61759696"/>
     
+    <xacro:property name="wheel_left_joint" value="left_wheel_joint"/>
+    <xacro:property name="wheel_right_joint" value="right_wheel_joint"/>
+
     <xacro:property name="wheels_separation" value="0.875"/>
     <xacro:property name="wheel_diameter" value="0.25"/>
 
+    <xacro:property name="wheel_acceleration" value="1.0"/>
+    <xacro:property name="wheel_torque" value="20"/>
+
     <xacro:property name="wheel_mass" value="4.45"/>
 
     <!-- Gazebo Motor -->
+    <!-- TODO: update parameters once they become known -->
+
     <xacro:property name="diff_drive_update_rate" value="30.0"/>
+
+    <xacro:property name="diff_drive_commandTopic" value="cmd_vel"/>
+    <xacro:property name="diff_drive_odometryTopic" value="odom"/>
+
+    <xacro:property name="diff_drive_odometryFrame" value="odom"/>
+    <xacro:property name="diff_drive_robotBaseFrame" value="base_link"/>
+
+    <xacro:property name="diff_drive_odometrySource" value="encoder"/>
+
+    <xacro:property name="diff_drive_publishWheelTF" value="false"/>
+    <xacro:property name="diff_drive_publishOdomTF" value="true"/>
+    <xacro:property name="diff_drive_publishTf" value="true"/>
+    <xacro:property name="diff_drive_publishWheelJointState" value="true"/>
+
+    <xacro:property name="diff_drive_legacymode" value="false"/>
+    <xacro:property name="diff_drive_rosDebugLevel" value="na"/>
+
+    <!-- Constants for hokuyo 2D LIDAR sensor -->
+    <xacro:property name="laser_visualize" value="false"/>
+    <xacro:property name="laser_update_rate" value="30"/>
+
+    <xacro:property name="laser_horizontal_samples" value="1080"/>
+    <xacro:property name="laser_horizontal_resolution" value="1"/>
+    <xacro:property name="laser_horizontal_min_angle" value="-2.356"/>
+    <xacro:property name="laser_horizontal_max_angle" value="2.356"/>
```

Comment:
it would better to define the min and max angle in terms of the `fov` (field of view - in degrees). Then in the `gazebo.xacro.urdf` you can compute it using `{+/- fov * pi / 180}` (note that pi is pre-defined in xacro: https://answers.ros.org/question/277011/using-pi-in-an-urdf-file/)